core/src/runtime/mod.rs

```diff
@@ -715,6 +716,22 @@ impl<'host> Runtime<'host> {
     self.execute()
   }
 
+  /// Execute an exported function using method selector. Does the same work as execute().
+  pub fn execute_selector(
```

Comment:
This sounds weird. It executes the selected method, not the selector :D
```suggestion
  pub fn execute_method(
```

Reply:
I made the names more explicit